Logos/WatchGuard_logo_Color.svg

```diff
@@ -0,0 +1 @@
56,6.31v8.77h-2.9V32.47c0-2.41-1-4.39-3.68-4.39a4.15,4.15,0,0,0-3.88,2.68A3,3,0,0,0,102,32v8.9h-2.9Z" transform="translate(-5.01 -2.8)"/><path class="cls-1" d="M133.1,40a21.35,21.35,0,0,1-6.84,1.13c-3.36,0-6.12-.8-8.29-2.72a10.14,10.14,0,0,1-3.1-7.65c0-6.12,4.58-10.6,12-10.6a14.89,14.89,0,0,1,5.57.94l-.73,2.16a11.87,11.87,0,0,0-4.9-.88c-5.4,0-8.92,3.11-8.92,8.26S121.26,39,126.46,39a9.84,9.84,0,0,0,3.81-.55V32.29H125.8V30.15h7.3Z" transform="translate(-5.01 -2.8)"/><path class="cls-1" d="M149.43,36.92c0,1.52,0,2.87.13,4H147l-.16-2.4h-.07a6.06,6.06,0,0,1-5.27,2.74C139,41.28,136,40,136,34.82V26.19h2.89v8.14c0,2.8,1,4.72,3.56,4.72a4.17,4.17,0,0,0,3.82-2.47,3.48,3.48,0,0,0,.26-1.37v-9h2.9Z" transform="translate(-5.01 -2.8)"/><path class="cls-1" d="M164.51,37.41a23.41,23.41,0,0,0,.23,3.53h-2.6l-.26-1.86h-.1a6,6,0,0,1-4.87,2.2c-3.23,0-4.87-2.1-4.87-4.24,0-3.56,3.42-5.52,9.58-5.48v-.31c0-1.22-.37-3.45-3.62-3.41a8.13,8.13,0,0,0-4.18,1.09l-.66-1.79a10.84,10.84,0,0,1,5.26-1.28c4.91,0,6.09,3.07,6.09,6Zm-2.83-4c-3.16-.07-6.74.45-6.74,3.32a2.45,2.45,0,0,0,2.69,2.56,4,4,0,0,0,3.92-2.47,2.55,2.55,0,0,0,.13-.85Z" transform="translate(-5.01 -2.8)"/><path class="cls-1" d="M167.87,30.79c0-1.73,0-3.23-.13-4.6h2.53l.13,2.9h.1A4.77,4.77,0,0,1,175,25.85a3.27,3.27,0,0,1,.78.1v2.5a10,10,0,0,0-1-.06,3.94,3.94,0,0,0-3.88,3.41,8.52,8.52,0,0,0-.1,1.28v7.86h-2.89Z" transform="translate(-5.01 -2.8)"/><path class="cls-1" d="M190.64,19.3V37.13c0,1.31,0,2.81.13,3.81h-2.6L188,38.38H188a5.92,5.92,0,0,1-5.39,2.9c-3.85,0-6.85-3-6.85-7.5,0-4.94,3.29-7.92,7.14-7.92a5.43,5.43,0,0,1,4.81,2.22h.06V19.3Zm-2.9,12.9a4.54,4.54,0,0,0-.13-1.16A4.21,4.21,0,0,0,183.43,28c-3,0-4.77,2.44-4.77,5.67,0,3,1.61,5.48,4.7,5.48a4.32,4.32,0,0,0,4.25-3.2,4.22,4.22,0,0,0,.13-1.15Z" transform="translate(-5.01 -2.8)"/><path d="M46.71,17.67a20.67,20.67,0,0,1,5.16,12.39s0,0,0,0a79.55,79.55,0,0,1-5,15.34s-2.94.12-8.11.19c-1.22-7.45-3.27-15.39-4.69-20.84H34c-1.42,5.45-3.47,13.39-4.69,20.84-5.17-.07-8.1-.19-8.1-.19-5.77-12.57-7-27.4-7-27.4,2.65-.19,4.88-.27,7.15-.32.56,5.56,2.51,16.57,3.61,19.71H25a83.48,83.48,0,0,1,5.43-19.92c2.58-.06,4.36-.1,7,0h.09A83,83,0,0,1,43,37.4h.09c1.11-3.14,3-14.15,3.62-19.71Z" transform="translate(-5.01 -2.8)"/><path class="cls-1" d="M33.85,5.48c-.9-.09-1.82-.14-2.76-.14a26,26,0,1,0,2.76.14l-.21,2.17a24,24,0,1,1-2.55-.13c.86,0,1.71.05,2.55.13Z" transform="translate(-5.01 -2.8)"/><path class="cls-2" d="M82,5.79a5.43,5.43,0,0,1-2.41,7.31l-15.1,7.62A5.45,5.45,0,0,1,59.62,11L74.73,3.38A5.44,5.44,0,0,1,82,5.79" transform="translate(-5.01 -2.8)"/><path class="cls-1" d="M195.42,20.83a1.62,1.62,0,1,1-1.61-1.57,1.59,1.59,0,0,1,1.61,1.57m-2.84,0a1.23,1.23,0,0,0,1.23,1.26,1.2,1.2,0,0,0,1.2-1.25,1.22,1.22,0,1,0-2.43,0m1,.82h-.36V20.07a4,4,0,0,1,.6-.05.8.8,0,0,1,.54.13.39.39,0,0,1,.17.34.41.41,0,0,1-.33.37v0c.15.05.23.17.28.39a1.51,1.51,0,0,0,.11.38h-.38a1.22,1.22,0,0,1-.13-.38c0-.17-.12-.25-.32-.25h-.18Zm0-.89h.17c.21,0,.37-.06.37-.23s-.11-.24-.34-.24l-.2,0Z" transform="translate(-5.01 -2.8)"/></svg>
```

Comment:
svg is not valid.
Logo has id and it is not GUID. Please refer
'https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md' for logo requirements. Please refer other logos available at https://github.com/Azure/Azure-Sentinel/tree/d7f381c61cb73891992d189b12cdd2e00d653d6f/Logos for reference.